cpp/tests/community/mg_louvain_helper.hpp

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
Are `mg_louvain_helper.[cu|hpp]` really specific to louvain? I'm just wondering if they're generally useful, should they be renamed and placed in the `cugraphtestutil` lib?

Reply:
They are currently specific to louvain.  I think they might be more generally useful.  But I wanted to have at least one more unit test that used it before moving into a more general location.

Reply:
And FYI,

renumber_ext_vertices
https://github.com/rapidsai/cugraph/pull/1443/files#diff-4106792213a4762c10634a44ac8c92f194994ec0b595dba4eccaa822e8353978R215

unrenumber_local_int_vertices
https://github.com/rapidsai/cugraph/pull/1443/files#diff-4106792213a4762c10634a44ac8c92f194994ec0b595dba4eccaa822e8353978R245

and unrenumber_local_int_vertices
https://github.com/rapidsai/cugraph/pull/1443/files#diff-4106792213a4762c10634a44ac8c92f194994ec0b595dba4eccaa822e8353978R279

will be added once PR 1443 gets merged.

There will be some overlaps and some functions here may become obsolete.